<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="alpha_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="theta_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="delta_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hundredHertz_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="twentymin_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="fourtyfive_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="userdesignated_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="select_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>SELECT</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>  1</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  2</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  3</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  4</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  5</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  6</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  7</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="down_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="up_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="power_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>POWER</string>
     </property>
    </widget>
    <widget class="QLabel" name="int_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>     INT</string>
     </property>
    </widget>
    <widget class="QLabel" name="canada_label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>270</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CANADA</string>
     </property>
    </widget>
    <widget class="QLabel" name="mindalive_label">
     <property name="geometry">
      <rect>
       <x>343</x>
       <y>27</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MIND ALIVE</string>
     </property>
    </widget>
    <widget class="QLabel" name="leftear_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  L</string>
     </property>
    </widget>
    <widget class="QLabel" name="rightear_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  R</string>
     </property>
    </widget>
    <widget class="QLabel" name="oasispro_label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="topleft_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bottomleft_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>MODE</string>
     </property>
    </widget>
    <widget class="QLabel" name="power_light">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>169</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectionButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Ear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>510</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Ear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wet earlobe (L)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wet earlobe (R)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
